### ISSUE TITLE
NbToggleModule not available

### ISSUE THREAD

**Issue by snandkule** [akveo/nebular#1943]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [ ] bug report
* [ ] feature request

### Issue description

@nebular/theme"' has no exported member 'NbToggleModule'
Not able to find NbToggleModule for  importing.
I think i am missing something silly.

**Current behavior:**

**Expected behavior:**

**Steps to reproduce:**

**Related code:**

```
insert short code snippets here
```

### Other information:

**npm, node, OS, Browser**
```
<!--
Node, npm: `node --version` and `npm --version`
OS: Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
Browser: Chrome/Safari/Firefox/etc?
-->
```

**Angular, Nebular**
```
<!--
Check your `package-lock.json` or locate a `package.json` in the `node_modules` folder.
-->
```

**Comment by nnixaa** [comment]
Hi @snandkule, please check your Nebular version, the toggle component is available starting 4.2.0.

**Comment by snandkule** [comment]
thanks :)
